packages/suite/src/utils/suite/oauth.ts

```diff
@@ -10,10 +10,6 @@ import { urlHashParams } from '@suite-utils/metadata';
 export const getOauthReceiverUrl = () => {
     if (!window.desktopApi) {
         const { origin } = window.location;
-        // For the purpose of e2e tests change the redirect url to develop branch on sldev.cz
-        if (origin.indexOf('sldev.cz') >= 0) {
-            return 'https://suite.corp.sldev.cz/suite-web/develop/wallet/static/oauth/oauth_receiver.html';
```

Comment:
imho this change needs to stay because this exact address is specified in Dropbox app setting, dropbox doesn't support wildcard addresses (like: ..sldev.cz/suite-web/branch-name-not-develop..)
sldev environment always should point to the address specified in Dropbox app. (due to postMessage origin in oauth.html file)

Reply:
well, yes... I was ok with it that labeling will not work in feature branches on sldev, but only in develop branch which is whitelisted. But yes, we may target it always to develop branch

Reply:
Ok, I put it back. Pros are all branches gonna work like this, cons that it is hardcoded and it may shoot someone in the leg one day. But probably worth it.